### PR TITLE
Change environment variables to use getenv

### PR DESCRIPTION
**Pull request by tractorcow**
Replaces #28

### CODE REVIEW

**Comment by ichaber** [comment]
Thanks for implementing the necessary changes @tractorcow 🌟

**Comment by chillu** [comment]
@tractorcow There's some PHPCS failures, could you have a look? Charlie is approving the changes (he's sitting next to me), so feel free to self merge - unless you think it needs someone else's review. This is critical for SSP 4.x operation, high priority

**Comment by tractorcow** [comment]
Ok fixed. Also moved to vendormodule.

File: composer.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
 		}
 	],
 	"require": {
+        "silverstripe/vendor-plugin": "^1.0",
```
Comment (sminnee) [inline composer.json]: Does this module expose front-end resources? If not, I don't think this module is a requirement.

Also, are we expecting that modules are allowed to rely on framework supplying this as a requirement, or do we expect that every module maintainer has to add this line? I'd suggest that relying on framework to properly take care of the implications of using silverstripe-vendormodule would be an acceptable division of responsibility.

Comment (tractorcow) [reply composer.json]: Ah good point, removed.

Modules without exposed resources don't need the plugin.

**Comment by chillu** [comment]
I've also published a 4.0.1 release: https://github.com/silverstripe/silverstripe-dynamodb/releases/tag/4.0.1, and adjusted the branch alias to match the main release line: https://github.com/silverstripe/silverstripe-dynamodb/pull/31

**Pull request merged**